addons/mail/static/src/models/thread_view/thread_viewer.js

```diff
@@ -77,6 +77,12 @@ function factory(dependencies) {
     }
 
     ThreadViewer.fields = {
+        /**
+         * true if the viewer is in a compact format, like in a chat window.
+         */
+        compact: attr({
```

Comment:
I prefer boolean fields being prefixed with `is`/`has`, just so it's clear it's a boolean.